R/h_bracket.R

```diff
@@ -0,0 +1,19 @@
+#' @rdname h_bracket
+#' @export
+setMethod(
+    f = "h_bracket",
+    signature = "character",
+    definition = function(x) {
+        if (nchar(x) >= 1) paste0("{\n", paste0(x, collapse = "\n"), "\n}\n")
+    }
+)
+
+#' @rdname h_bracket
+#' @export
+setMethod(
+    f = "h_bracket",
+    signature = "list",
+    definition = function(x) {
+        paste0(paste0(x, collapse = "\n"), "\n")
+    }
+)
```

Comment:
If you are collapsing `priors` into strings and combining them with `model` is this function/method still needed?

Reply:
I explain my motivation for the S4 further below.